modules/op-signing-secrets-in-tekton-chains.adoc

```diff
@@ -39,6 +39,54 @@ Cosign prompts you for a password, and creates a Kubernetes secret.
 
 . Store the encrypted `cosign.key` private key and the `cosign.password` decryption password in the `signing-secrets` Kubernetes secret. Ensure that the private key is stored as an encrypted PEM file of the `ENCRYPTED COSIGN PRIVATE KEY` type.
 
+
+[id="chains-signing-secrets-skopeo_{context}"]
+== Signing using skopeo
+
+You can generate keys using skopeo and use them in the `cosign` signing scheme with {tekton-chains}:
+
+. Install link:https://github.com/containers/skopeo[skopeo].
+
+. Generate a public/private key pair.
++
+[source,terminal]
+----
+$ skopeo generate-sigstore-key --output-prefix <mykey>
+----
++
+Replace `<mykey>` with a key name of your choice. Skopeo prompts you for a passphrase for the private key. It then creates the key files named `<mykey>.private` and `<mykey>.pub`.
+. Encode the key files using base64.
++
+[source,terminal]
+----
+$ base64 <mykey>.pub > b64.pub
+$ base64 <mykey>.private > b64.private
+----
++
+. Encode the passhprase using base64.
++
+[source,terminal]
+----
+$ echo -n '<passphrase>' | base64 > b64.passphrase
+----
++
+. Edit the `signing-secrets` secret in your {pipelines-title} namespace.
++
+[source,terminal]
+----
+$ oc edit secret -n <pipelines-namespace> signing-secrets
```

Comment:
The secret doesn't exist on fresh installation and `oc edit` will fail with `Error from server (NotFound): secrets "signing-secrets" not found`. If it existed, it would probably contain some data already and it would fail with error `data: Forbidden: field is immutable when 'immutable' is set`

There are multiple options but the easiest one is to
1. run `oc create secret generic signing-secrets -n openshift-pipelines` before `oc edit`
2. update `-n <pipelines-namespace>` to `-n openshift-pipelines` as our customers have to use this namespace

Reply:
Fixed